Overview.src.html

```diff
@@ -180,6 +180,15 @@ <h2 class="no-num no-toc">Table of Contents</h2>
 
 <p><dfn>Fullscreen is supported</dfn> if there is no previously-established user
 preference, security risk, or platform limitation.
+
+<p>An algorithm is <dfn>allowed to request fullscreen</dfn> if one of the following conditions are
+true:
```

Comment:
"if one of the following is true"

Also this might need to be wrapped per 100column rule.